pages/common/spatial.md

```diff
@@ -0,0 +1,27 @@
+# spatial
+
+> A set of commands for managing and developing SpatialOS projects.
+
+- Build workers for local deployment on Unity on Windows:
+
+`spatial worker build --target=development --target=Osx`
+
+- Build workers for local deployment on Unreal:
+
+`spatial worker build --target=local --target=Windows`
+
+- Deploy locally:
+
+`spatial local launch {{launch_config}} --snapshot={{snapshot_file}}`
+
+- Launch a local worker to connect to your local deployment:
+
+`spatial local worker launch UnityClient default`
```

Comment:
Should UnityClient be in tokens `{{}}` ?

Reply:
UnityClient (as it is right now) is the default, but I can make it more generic and have it in {{}}